fluent/src/resolver.js

```diff
@@ -123,6 +164,15 @@ function Tags(env, {name}) {
 /**
  * Resolve a variant expression to the variant object.
  *
+ * @param   {Object} env
+ *    Resolver environment object.
+ * @param   {Object} expr
+ *    An expression to be resolved.
+ * @param   {String} expr.id
+ *    An ID of a message for which the variant is resolved.
```

Comment:
Looking at the parser, I realize now that `id` is a `type: ref` object, with a `name` field.